examples/analytics-3p-remote-frame.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Requests Frame</title>
+    <script>
+      /**
+       * To receive analytics events in a third-party frame, you must
+       * implement this method, with this signature. Within that method, you
+       * must create a function which processes the analytics events, and
+       * pass that function to the registerAmpAnalyticsEventListener() method
+       * of the object which is passed as a parameter to
+       * onNewAmpAnalyticsInstance().
+       * @param ampAnalytics  Call registerAmpAnalyticsEventListener() on this,
+       * passing your function which will receive the analytics events.
+       */
+      window.onNewAmpAnalyticsInstance = ampAnalytics => {
+        ampAnalytics.registerAmpAnalytics3pEventsListener(events => {
+          events.forEach(event => {
+            // Now, do something meaningful with the AMP Analytics event
+            console.log("Received an event: " + event.ampAnalytics3pEvent +
+              " from: " + event.senderId + ", and my extra data is: " +
+              ampAnalytics.getExtraData(event.senderId));
+            ampAnalytics.sendMessageToCreative(
+              {'status': 'received', 'somethingElse': '42'});
+          });
+        });
+        // The following is optional:
+        ampAnalytics.registerAmpAnalytics3pExtraDataListener(extraDataMsgs => {
```

Comment:
As mentioned, we should remove the registerAmpAnalytics3pExtraDataListener and instead you should either pass the initial state in onNewAmpAnalyticsInstance (e.g. (ampAnalytics, startUpState)) or make it a field of the ampAnalytics instance such that it has 1 field (startUpState) and 2 functions (onAmpAnalytics3pEvents & sendMessageToCreative)